Cohort1/Medlista/build.gradle.kts

```diff
@@ -5,9 +5,9 @@ plugins {
     //trick: for the same plugin versions in all sub-modules
     id("com.android.application").version("7.3.1").apply(false)
     id("com.android.library").version("7.3.1").apply(false)
-    kotlin("android").version("1.7.10").apply(false)
-    kotlin("multiplatform").version("1.7.10").apply(false)
-    //id("org.jetbrains.kotlin.android") version "1.6.10" apply false
+    kotlin("android").version("1.8.10").apply(false)
+    kotlin("multiplatform").version("1.8.10").apply(false)
+    id("io.realm.kotlin") version "1.6.0"
```

Comment:
Thanks for updating the versions! If I have any issues with the newer versions I'll let you know.